arctic/store/_ndarray_store.py

```diff
@@ -145,9 +143,6 @@ def _index_range(self, version, symbol, from_version=None, **kwargs):
         """
         from_index = None
         if from_version:
-            if version['base_sha'] != from_version['base_sha']:
```

Comment:
Ah, that sounds familiar. On master I was looking at the write (which sets this field anew) and the append which copies this, and was worried that this was protecting a collision between the two method calls.  It's quite an unlikely issue anyhow.